pages/your_domain/human_data.md

```diff
@@ -91,21 +86,22 @@ For human data, it is very important to use technical and procedural measures to
   - What information security measures that need to be established should be defined at the planning stage (see above), when doing a risk assessment, e.g. a GDPR Data Protection Impact Assessment. This should identify information security risks, and define measures to mitigate those risks.
   - Contact the IT or Information security office at your institution to get guidance and support to address these issues.
   - [ISO/IEC 27001](https://en.wikipedia.org/wiki/ISO/IEC_27001) is an international information security standard adopted by data centres of some universities and research institutes.
-* Locating tools and platforms suited to handle human data
+* Check whether there are local/national tools and platforms suited to handle human data.
   - Local research infrastructures might have established compute and/or storage solutions with strong information security measures tailored for working on human data (see some examples below). Contact your institute or your ELIXIR node for guidance.
     - Denmark - [Computerome](https://computerome.dk/)
     - Finland - [ePouta](https://research.csc.fi/-/epouta), which is part of the [CSC Tools Assembly](csc_assembly)
     - Norway - [TSD Tools Assembly](tsd_assembly)
     - Sweden - [Bianca](https://www.uppmax.uu.se/resources/systems/the-bianca-cluster/)
     - Spain - [MareNostrum](https://www.bsc.es/marenostrum/access-to-supercomputing-resources)
+    - Luxembourg - [TransMed Tools Assembly](transmed)
```

Comment:
I am just not sure if we should list national stuff on the domain page ...

Reply:
I would agree. I think users should be directed towards the national pages where this type of information should be reported (along with national policies).

Reply:
Good Idea @bianchini88 I will refer to each national page here.

Reply:
I instead referred to the RDMkit national resources index page. As countries other than those in the bulleted list also have resources for sensitive data.